src/docs/event.yaml

```diff
@@ -0,0 +1,316 @@
+tags:
+ - name: create a new event
+    description: create a new event
+paths:
+  /api/events:
+    post:
+      tags:
+        - event
+      summary: create a new event
+      description: create a new event
+      consumes:
+        - multipart/form-data
+      produces:
+        - application/json
+      requestBody:
+        required: true
+        content:
+          multipart/form-data:
+            schema:
+              $ref: "#/components/schemas/newEvent"
+      responses:
+        201:
+          description: Authenticated user succesfully created event.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Event"
+        400:
```

Comment:
I can't see where your endpoint has this error. you probably should remove this.